libs/application/templates/children-residence-change/src/dataProviders/MockParentNationalRegistryProvider.ts

```diff
@@ -14,27 +14,17 @@ export class MockParentNationalRegistryProvider extends BasicDataProvider {
 
   async provide(application: Application): Promise<PersonResidenceChange> {
     const crcApplication = (application as unknown) as CRCApplication
-    return (
-      crcApplication.answers?.mockData?.parentNationalRegistry?.data ||
-      this.handleError()
-    )
-  }
-  handleError() {
-    return Promise.resolve({
-      id: '1',
-      name: 'Eiríkur Jónsson',
-      ssn: '120486-7899',
-      address: 'Suðurgata 35, íbúð 2',
-      postalCode: '105',
-      city: 'Reykjavík',
-    })
+    const data = crcApplication.answers?.mockData?.parentNationalRegistry?.data
+    if (data && data.ssn && data.ssn !== '') {
+      return data
+    }
+    throw new Error('Ekki tókst að ná í gögn eða kennitölu vantar')
```

Comment:
Should we turn this around? And return the data at the bottom? 
I usually like that structure better. Edge cases in the middle and default value in the end?